diffkemp/diffkemp.py

```diff
@@ -580,3 +587,164 @@ def text_indent(text, width):
             if not output_dir:
                 output.write(text_indent("}}}\n", indent - 2))
             output.write("\n")
+
+
+class YamlOutput:
```

Comment:
For now i have moved it to file `output.py` .

Reply:
> I'd prefer moving this class into a separate module (source file), e.g. `output.py`.
> 
> Another option would be to transform this into a `Result.to_yaml()` method but I'm not sure if it would logically fit that. @lenticularis39 WDYT?

I think it makes sense to have it in `output.py`, we also have other formatting code like `print_syntax_diff` that can be moved into there.